todo.md

```diff
@@ -0,0 +1,4 @@
+- add `.fill` to fill paths
+- add `.arc`, `.arcTo`
+- add `fillText`
+- think up more elegant way to call `.frame()`
```

Comment:
i'm not sure how can we hide the differences between node and the browser, or even that we can hide them completly